### ISSUE TITLE
Unable to move light using G hotkey in Blender 3.1.0

### ISSUE THREAD

**Issue by leomoon** [leomoon-studios/blender-light-studio#65]
move:
```
Python: Traceback (most recent call last):
  File "C:\Users\LeoMoon\AppData\Roaming\Blender Foundation\Blender\3.1\scripts\addons\leomoon-lightstudio\operators\modal.py", line 243, in invoke
    self.mouse_x = context.area.width/2
  File "D:\LeoMoon\Desktop\blender-3.1.0-windows-x64\3.1\scripts\modules\bpy_types.py", line 778, in __setattr__
    return setattr(properties, attr, value)
TypeError: bpy_struct: item.attr = val: LIGHT_STUDIO_OT_grab.mouse_x expected an int type, not float

location: <unknown location>:-1
```
rotate:
```
Python: Traceback (most recent call last):
  File "C:\Users\LeoMoon\AppData\Roaming\Blender Foundation\Blender\3.1\scripts\addons\leomoon-lightstudio\operators\modal.py", line 73, in invoke
    self.mouse_x = context.area.width/2
  File "D:\LeoMoon\Desktop\blender-3.1.0-windows-x64\3.1\scripts\modules\bpy_types.py", line 778, in __setattr__
    return setattr(properties, attr, value)
TypeError: bpy_struct: item.attr = val: LIGHT_STUDIO_OT_rotate.mouse_x expected an int type, not float

location: <unknown location>:-1
```
scale:
```
Python: Traceback (most recent call last):
  File "C:\Users\LeoMoon\AppData\Roaming\Blender Foundation\Blender\3.1\scripts\addons\leomoon-lightstudio\operators\modal.py", line 156, in invoke
    self.mouse_x = context.area.width/2
  File "D:\LeoMoon\Desktop\blender-3.1.0-windows-x64\3.1\scripts\modules\bpy_types.py", line 778, in __setattr__
    return setattr(properties, attr, value)
TypeError: bpy_struct: item.attr = val: LIGHT_STUDIO_OT_scale.mouse_x expected an int type, not float

location: <unknown location>:-1
```
Also clicking alight in LightPanel will close it.

**Comment by deadman49** [comment]
same problem, not working on 3.1

**Comment by leomoon** [comment]
Fixed in LightStudio 2.11.0
